common/changes/@microsoft/api-documenter/vibraga-experimentalDocumenter_adjustments_2019-05-24-22-43.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-documenter",
+      "comment": "Add new `nonEmptyCategoryNodeNames` config option to the ExperimentalYamlDocumenter. Refactor filter logic. Add documentation and modify the test package.",
```

Comment:
This comment is describing implementation details.  Normally a change log should be written for an audience of people who use the tool (e.g. what feature did you add? what bug did you fix?), not people who work on the tool.

But since `ExperimentalYamlDocumenter` is experimental and not intended to be used by external people yet, I'd just say something simple like `Improved the experimental "api-documenter generate" command`.

Reply:
Fixed!